build.gradle

```diff
@@ -102,6 +102,17 @@ jacocoTestReport {
     }
 }
 
+configurations.all {
```

Comment:
This is a bit ugly, as we need to do it in all upstream repos, but I can't really think of a better way.

Reply:
Yeah, I've tried a number of different options, but all of them look almost as bad. One possible alternative is to repackage artifacts, but this has significant overhead and therefore has sense to bigger teams only.